beetmoverscript/constants.py

```diff
@@ -76,10 +76,13 @@
 
 # actions that imply actual releases, hence the need of `build_number` and
 # `version`
-RELEASE_ACTIONS = (
+PROMOTION_ACTIONS = (
     'push-to-candidates',
     'push-to-releases',
```

Comment:
Shoudn't we remove `push-to-releases` from here and only keep it in RELEASE_ACTIONS? Otherwise, we'll have both `is_release_action` and `is_promotion_action` return True for same action scope.

Reply:
Sure, then I would have to change https://github.com/mozilla-releng/beetmoverscript/pull/89/files#diff-14bc831c46930a709ba12c8fb4f46eb9R88 to check for both. Doable.